plugins/exec-all.yaml

```diff
@@ -4,20 +4,20 @@ metadata:
   name: exec-all
 spec:
   platforms:
-  - uri: "https://github.com/jpdasma/kubectl-exec-all/archive/v1.0.0.zip"
-    sha256: "93f4da0b9127833924ede21d1ac1452a189e23b54339d8eeac1623a89dbfbd17"
+  - uri: "https://github.com/jpdasma/kubectl-exec-all/archive/v1.1.0.zip"
+    sha256: "da3158fc5c88821c27f4534e4cc059383b3365e46421d84502d7430d7f817cc4"
     bin: exec-all.sh
     files:
     - from: "/*/unix/*"
       to: "."
     selector:
       matchExpressions:
       - {key: os, operator: In, values: [darwin, linux]}
-  version: "v1.0.0"
+  version: "v1.1.0"
   shortDescription: "Execute a command in all of the containers in a specified resource"
   caveats: |
     Usage:
-      kubectl plugin exec-all <RESOURCE_TYPE> <RESOURCE_NAME> -- <COMMAND>
+      kubectl exec_all <RESOURCE_TYPE> <RESOURCE_NAME> -- <COMMAND>
```

Comment:
you can keep this as `kubectl exec-all`. Both will work. Krew will automatically symlink your binary as kubectl-exec_all.